action.yaml

```diff
@@ -57,6 +57,8 @@ runs:
     - id: init
       shell: bash
       run: |
+        echo "LACEWORK_START_TIME=$(date --rfc-3339=seconds)" >> $GITHUB_ENV
+        echo "LACEWORK_CONTEXT_ID=$(echo $RANDOM | md5sum | head -c 32)" >> $GITHUB_ENV
```

Comment:
Is this used as key to combine all the telemetry data coming from the same run?

Reply:
Yes, exactly.

Reply:
Could we use something like the run id for this?

Reply:
I see. Thanks for the clarification.